nevergrad/optimization/optimizerlib.py

```diff
@@ -11,6 +11,7 @@
 import numpy as np
 from bayes_opt import UtilityFunction
 from bayes_opt import BayesianOptimization
+from typing import List
```

Comment:
```suggestion
```

For consistency, please your `tp.List` instead of `List` (`typing` is already imported as `tp`)

Reply:
Done